galite-core/src/main/kotlin/org/kopi/galite/form/VImageField.kt

```diff
@@ -83,7 +83,7 @@ class VImageField(val bufferSize: Int, val iconWidth: Int, val iconHeight: Int)
   /**
    * Returns the search conditions for this field.
    */
-  override fun getSearchCondition(): (Expression<*>.() -> Op<Boolean>)? = null
+  override fun <T> getSearchCondition(column: ExpressionWithColumnType<T>?): Op<Boolean>? = null
```

Comment:
do we need to change this?

Reply:
yes it is more correct to do that, already checked with Hichem